MapboxNavigation/Turf.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import Mapbox
+import Turf
+
+extension Turf.BoundingBox {
+    init(coordinateBounds: MGLCoordinateBounds) {
+        self.init(coordinateBounds.sw, coordinateBounds.ne)
```

Comment:
I somehow missed that the BoundingBox initializer has [unlabeled arguments](https://github.com/mapbox/turf-swift/blob/87d12f6c7a9ca28d451c1f85fe3a811d72485b50/Sources/Turf/BoundingBox.swift#L25). We should fix that independently of this PR.

Reply:
This was fixed in mapbox/turf-swift#132, which will be coming in v2._x_.